capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js

```diff
@@ -132,7 +199,9 @@ function createContentString(name, items) {
 
                 </div>
                 <div id="posts" class="tab-pane fade">
-                    <h4>This is where the posts for ${name} will go.</h4>
+                    <h5>There are no posts for ${name} yet. Click
+                    <a href="../pages/comments.jsp"><u>here</u></a>
```

Comment:
Is there a reason for the underline tag? I think a tag's by default underline the hyperlink